deno/lib/types.ts

```diff
@@ -45,13 +45,55 @@ import {
 export type RefinementCtx = {
   addIssue: (arg: IssueData) => void;
   path: (string | number)[];
+  params?: any;
```

Comment:
this should be Input ?

Reply:
No this is a parameter object added by the PR to the parse method, the idea of this parameter is to customization to refinement based on data available at runtime during the parsing. I typed it as any as typing it would mean to add a param generic type to ZodType and propagate it across different implementations of ZodType, it would be great but I didn't add time to do that yet. It would be a subject for a future PR.